src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+export const PeopleList: React.FC = () => {
+  const [isLoading, setIsLoading] = React.useState(true);
```

Comment:
It;s extra, just import it from React
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```